terraform-tests/mssql_test.go

```diff
@@ -52,9 +54,8 @@ var _ = Describe("mssql", Label("mssql-terraform"), Ordered, func() {
 		It("should create a db instance with the right values", func() {
 			Expect(AfterValuesForType(plan, "aws_db_instance")).To(MatchKeys(IgnoreExtras, Keys{
 				"engine":               Equal("sqlserver-ee"),
-				"engine_version":       Equal("15.00.4236.7.v1"),
 				"identifier":           Equal("csb-mssql-test"),
-				"instance_class":       Equal("db.m6i.xlarge"),
+				"instance_class":       Equal(""),
```

Comment:
I believe we usually take the approach of adding some value to fields that are required, specially because in this scenario it is required by the provider as well.  But this is technically more correct, so no real objection.